### PR TITLE
Issue18 request for comment

### PR DESCRIPTION
**Pull request by tkphd**
Addresses #18.

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/awesome-nist/22)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,27 +1,146 @@
...
+[checklist](AWESOME.md) into the comment field, and check off
+whatever applies. Don't worry if several boxes go unchecked:
+it gives the developers room to grow!
+
+## Guidelines for Contributors
+
+To add a repository to the curated list, please
+
+* Locate the [Issue][_issu] in which the repository was nominated,
+  or create a new Issue if one does not already exist.
```
Comment (guyer) [inline CONTRIBUTING.md]: What function does an issue serve that isn't handled by just creating the pull request?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,27 +1,146 @@
...
+### Required Files
+
+Developers signed the [Rules of Behavior][_grob], which requires both
+LICENSE.md (or similar) and README.md (or similar).
+
+#### LICENSE.md
+
+LICENSE.md must contain the official [dedication to the public domain][_pub].
+Assignments or reservations of copyright, *e.g.* under [BSD][_bsd], [GPL][_gpl],
+or [MIT][_mit] licensing terms, are unlawful per [17 U.S.C. §105][_copy].
```
Comment (guyer) [inline CONTRIBUTING.md]: - Projects done in collaboration with external partners? 
 - Projects conducted by non-Federal staff, e.g., contractors, FGRs, etc.?

**Comment by tkphd** [comment]
Good feedback, @guyer, thank you.

@agmorey, can you weigh in on the licensing requirements? The GitHub ROB leaves no room for copyright/licenses, which implies that copyrighted works should be hosted outside the usnistgov organization, but I might be reading that too narrowly.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -19,14 +19,10 @@ it gives the developers room to grow!
 
 To add a repository to the curated list, please
 
-* Locate the [Issue][_issu] in which the repository was nominated,
```
Comment (guyer) [inline CONTRIBUTING.md]: My initial reaction was that it's *OK* if there's an issue, and if so, we should reference it, but the more I think about it, for this use case, I think if somebody thinks highly of a project, they should just do a pull request and not (effectively) suggest that somebody else should do a pull request.

tl;dr: I approve of removing all discussion of issues

**Review by guyer — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (tkphd) [reply CONTRIBUTING.md]: Yeah, in thinking about your question, that was my conclusion, too.

Merging. @agmorey and @mnewrock, if you have objections, please file issues.
